etc/ci/update-wpt-checkout

```diff
@@ -15,6 +15,8 @@ BRANCH_NAME="wpt_update_${CURRENT_DATE}"
 
 export GIT_AUTHOR_NAME="WPT Sync Bot"
 export GIT_AUTHOR_EMAIL="josh+wptsync@joshmatthews.net"
+export GIT_COMMITTER_NAME="${GIT_AUTHOR_NAME}
```

Comment:
nit: needs trailing double quote

Reply:
Also, I didn't know about these environment variables, this is a nice trick for setting these in a script.

Reply:
Haha yes what a nit

Reply:
small but important :) this is r=me after that's addressed.